source/atlas-cli-changelog.txt

```diff
@@ -12,6 +12,82 @@
    :depth: 1
    :class: singlecol
 
+.. _atlas-cli-1.1.5:
+
+{+atlas-cli+} 1.1.5
+-------------------
+
+Released 2 August 2022
+
+- Fixes a bug where telemetry tracking events were not propagated when 
+  a default profile was not set.
+
+- Fixes a bug where it was not possible to 
+  filter organizations or projects by name during onboarding.
+
+- Fixes a bug where some options were not validated on command 
+  :ref:`atlas-backups-restores` job start.
+
+- Improves documentation and adds examples to 
+  :ref:`atlas-privateEndpoints-aws-describe`.
+
+- Improves onboarding for users with more than 500 organizations.
+
+- Improves the message when an operation completes in a free 
+  cluster, directing users to an upgrade command.
```

Comment:
I think this is what was meant by the original sentence but wasn't sure, feel free to change if needed
```suggestion
  cluster. The message now directs users to an upgrade command.
```

Reply:
That's my understanding too